### DISCUSSION TITLE
Can I change the generated Link to not include an ID #529

### DISCUSSION THREAD

**Discussion by Felitendo** [OpnForm/OpnForm#530] [Q&A] [answered]
Right now when I complete a Forms Setup it gives me a link like this:

http://forms.domain.tld/forms/some-title-le4iyv

Can I change this to generate a link like this?:

http://forms.domain.tld/forms/some-title

**Comment by JhumanJ** [comment, marked as answer]
Hi there, this is now possible by default in the self-hosted version!
![CleanShot 2025-05-20 at 20 30 42](https://github.com/user-attachments/assets/ddba01fb-9642-4d06-995e-b16e368661b9)

**Reply by Felitendo** [reply]
Nice, thank you very much
